test/unit/supervisor/pod-watch-handler-caches.test.ts

```diff
@@ -47,14 +47,14 @@ tap.test('image and workload image cache', async (t) => {
   const podObject: V1Pod = YAML.parse(podFixture);
   await pod.podWatchHandler(podObject);
   await sleep(500);
-  t.equals(pushCallCount, 2, 'pushed data to send');
+  t.equals(pushCallCount, 1, 'pushed data to send');
```

Comment:
Can you Jestify the tests 🙏 
Based on our discussion, you touched it, you jest it :)

Reply:
Updated it.

I considered just deleting these tests because they're kinda pointless but whatever